### PR TITLE
/enterprise url with enterprise ethereum information and resources

### PR DESCRIPTION
**Pull request by faraggi**
There are 3 main changes proposed in this PR:

- added an `/enterprise` page path with information on enterprise ethereum
- added a enterprise intro-block section to the homepage with links to /enterprise resources
- added an "Enterprise Ethereum" h2+section on the `/learn` page with a link to the /enterprise page

I decided not to change the styling on the homepage intro-section (ie. to center align the block), but this could be done if needed.

### CODE REVIEW

**Comment by virgil** [comment]
Looks like a reasonable v1 to me.

File: docs/enterprise/index.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+For more in depth information, here are some related articles:
+
+- [5 Reasons Why Enterprise Ethereum Is so Much More Than a Distributed Ledger Technology](https://media.consensys.net/5-reasons-why-enterprise-ethereum-is-so-much-more-than-a-distributed-ledger-technology-c9a89db82cb5)
+- [Blockchain Use Cases and Applications by Industry](https://media.consensys.net/enterprise-ethereum-blockchain-use-cases-and-applications-by-industry-3914d1210049)
+- [EY releases zero-knowledge proof blockchain transaction technology to the public domain to advance blockchain privacy standards](https://www.ey.com/en_gl/news/2019/04/ey-releases-zero-knowledge-proof-blockchain-transaction-technology-to-the-public-domain-to-advance-blockchain-privacy-standards)
+- [Introduction to Quorum: Blockchain for the Financial Sector](https://medium.com/blockchain-at-berkeley/introduction-to-quorum-blockchain-for-the-financial-sector-58813f84e88c)
+
+
+## Organisations
```
[the remark this answers is not in the document]
Comment (faraggi) [reply docs/enterprise/index.md]: Sure, no problem.
I'm used to seeing both so neither shocked me, but I agree using both is like -gasp - mixing tabs and spaces!

**Comment by faraggi** [comment]
@jjmstark  @samajammin 

> * The language should be changed to reflect the fact that "enterprise Ethereum" doesn't mean just private networks. This is particularly relevant for the Mainnet Working Group. Instead, we should focus on Enterprise _needs_ like privacy and permissioning.

Did my best to reflect that with 2 changes (intro div, why enterprise? last bullet, 

>   * Change "Enterprise Focused Projects" to "Enterprise Clients"
>   * add Autonity under Clearmatics, and Besu under PegaSys

My thoughts were to first list Enterprise Focused Projects that _provide blockchain services_.
The next heading, `Protocol and Infrastructure` is about clients and protocol level tools, where Autonity is already listed.
I've changed `Enterprise Focused Projects` to `Enterprise Focused Services` in order to clarify.
Let me know if that's clear.

> * "Also curious how Sawtooth relates to Enterprise Ethereum? It’s an entire blockchain platform unto itself"

This is directly from their docs: 


> Hyperledger Sawtooth is an enterprise blockchain platform for building distributed ledger applications and networks. The design philosophy targets keeping ledgers distributed and making smart contracts safe, particularly for enterprise use.[1]



[1] https://sawtooth.hyperledger.org/docs/core/releases/latest/introduction.html

File: docs/enterprise/index.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+- [PegaSys Plus](https://pegasys.tech/enterprise/) *offers the same features and functionalities as HF Besu, plus additional enterprise focused benefits*
+- [Quorum](https://www.goquorum.com/) *open source blockchain platform that combines the innovation of the public Ethereum community with enhancements to support enterprise needs*
+
+## Protocol and Infrastructure
+
+- [Hyperledger Besu](https://www.hyperledger.org/projects/besu) *open-source Ethereum client developed under the Apache 2.0 license and written in Java*
+- [Hyperledger Burrow](https://kaleido.io/) *modular blockchain client with a permissioned smart contract interpreter partially developed to the specification of the Ethereum Virtual Machine (EVM)*
+- [Infura](https://infura.io/) *scalable API access to the Ethereum and IPFS networks*
+- [Kaleido](https://kaleido.io/) *full-stack platform for building and running cross-cloud, hybrid enterprise ecosystems*
+- [Hyperledger Sawtooth](https://www.hyperledger.org/projects/sawtooth) *modular platform for building, deploying, and running distributed ledgers.*
```
Comment (samajammin) [inline docs/enterprise/index.md]: I recommend removing this for now until it's clear this is an Ethereum project. We want to avoid creating confusion.

Comment (faraggi) [reply docs/enterprise/index.md]: I'm sure its an Enterprise Ethereum project, but I've gone ahead and removed the mentions for now.

File: docs/enterprise/index.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
+---
+title: Enterprise
```
[the remark this answers is not in the document]
Comment (faraggi) [reply docs/enterprise/index.md]: great idea. done.

[the remark this answers is not in the document]
Comment (samajammin) [reply docs/enterprise/index.md]: Minor formatting issue that's causing a [build error on this branch](https://app.netlify.com/sites/ethereumorg/deploys/5d7c8eb9ee1efd0009831932) - my example may have led you astray. [Proper example here](https://vuepress.vuejs.org/guide/frontmatter.html#predefined-variables). This should work:
```
---
title: Enterprise
meta:
    - name: description
      content: Guides, articles, and tools about public and private Ethereum blockchains for enterprise
    - property: og:title
      content: Enterprise Ethereum
    - property: og:description
      content: Guides, articles, and tools about public and private Ethereum blockchains for enterprise
lang: en-US
sidebar: auto
sidebarDepth: 0
---
```

**Comment by faraggi** [comment]
That should've fixed it.
Checks are passing now, deploy successful.

**Comment by jjmstark** [comment]
One last change: on the homepage, let's change the title of that section to simply say "Enterprise" instead of "Enterprise Ethereum" to make it consistent with the other subpage headers.

**Comment by faraggi** [comment]
@jjmstark Done.

**Pull request merged**